test/test_core.py

```diff
@@ -5989,11 +5989,11 @@ def test_unistd_sysconf_phys_pages(self):
   def test_unistd_unlink(self):
     # symlinks on node.js on non-linux behave differently (e.g. on Windows they require administrative privileges)
     # so skip testing those bits on that combination.
+    if any([arg in self.emcc_args for arg in ('-DNODEFS', '-DNODERAWFS')]) and MACOS:
+      self.skipTest('only tested on linux')
```

Comment:
Can you also add `@crossplatform` so that it would fail without this change in our CI?

Reply:
I think can you elide the square brackets here.

Also, I think it might be easier to read if `MACOS` came before the any statement.

Reply:
Maybe, but it's a little bit silly to add @crossplatform to a test that only works on one platform?

Reply:
(this will fail on Chromium CI without the change, that's how I noticed it)

Reply:
Right.. I just want to make sure we could also get the same failure here.  

We can then work to remove the skipping part over time (perhaps).

Reply:
Done.